tests/injection/outjection.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+# SPDX-License-Identifier: MPL-2.0
+
+from typegraph import typegraph, Policy, t, Graph
+from typegraph.runtimes import RandomRuntime, DenoRuntime
+
+
+@typegraph()
+def outjection(g: Graph):
+    deno = DenoRuntime()
+    random = RandomRuntime(seed=1)
+
+    g.expose(
+        Policy.public(),
+        randomUser=deno.identity(t.struct()).extend(
```

Comment:
tangent: would it be sensible to move `extend` onto `t.Struct`?

Reply:
`extend` was originally on `t.struct`, but it is currently missing on the TS SDK. On `t.func`, it applies on the output type (if it is a `t.struct`). Main reason was the heavy use of `deno.identity` in the tests, which btw won't play well with injections after this PR...

Reply:
https://linear.app/metatypedev/issue/MET-778/move-extend-to-tstructand-documentation